lib/wallet/http.js

```diff
@@ -1317,7 +1317,14 @@ class HTTP extends Server {
       res.setHeader('From-Cache', fromCache);
 
       if (broadcast) {
-        const tx = await req.wallet.sendMTX(mtx, passphrase);
+        const blockingSend = (bid <= 10_000_000);
```

Comment:
nit: booleans should start with `is`, `has`, `does`, etc

Reply:
fixed ...